ERCS/erc-7730.md

```diff
@@ -1380,6 +1380,9 @@ In the future, this specification could be extended to structured data like Meta
 
 The `interpolatedIntent` field is particularly well-suited for [EIP-5792](./eip-5792.md) batch transactions, as it enables wallets to concatenate multiple operation descriptions into a single, natural language sentence. By joining individual `interpolatedIntent` strings with " and ", wallets can provide users with clear, readable summaries of complex multi-step operations like "Approve Uniswap Router to spend 1000 USDC and Swap 1000 USDC for at least 0.25 WETH". This significantly improves the user experience when reviewing batch transactions compared to displaying separate, disconnected operation descriptions. See the [Value Interpolation](#value-interpolation) section for detailed implementation guidance and examples.
 
+### Common Keywords
+The DeFi ecosystem has matured and has some repeating usecases with well defined terms for different actions. While this ERC does not define specific standards for DEX or Lending protocol intents, for the sake of easier user readibility, teams with similar products should align on similar terminology to describe actions and actors. We encourage the creation of another layer of standartization on top of ERC-7730 to be recommeneded and even enforced in clear signing registries for improved user security.
```

Comment:
A real-world example would be helpful here IMO.

Reply:
I didn't want to mean a detailed example but more like: 

> ...well defined terms for different actions e.g. using "Swap" for all similar actions

but just a small suggestion. Feel free to include or not :)